Libraries/Blob/URL.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2015-2016 Silk Labs, Inc.
+ * All Rights Reserved.
+ *
+ * @providesModule URL
+ * @flow
+ */
+'use strict';
+
+const Blob = require('Blob');
+const { BlobModule } = require('react-native').NativeModules;
+
+let BLOB_URL_PREFIX = null;
+
+if (typeof BlobModule.BLOB_URI_SCHEME === 'string') {
+  BLOB_URL_PREFIX = BlobModule.BLOB_URI_SCHEME + ':';
+  if (typeof BlobModule.BLOB_URI_HOST === 'string') {
+    BLOB_URL_PREFIX += `//${BlobModule.BLOB_URI_HOST}/`;
+  }
+}
```

Comment:
What if they are not strings? Is that an expected state or an error? If it's an error consider logging an error rather than failing silently.